docs/source/best-practices/checkpointing.rst

```diff
@@ -7,7 +7,7 @@ The complexity of HPC systems may introduce unpredictable behaviors and may resu
 Checkpointing will allow you to:
 
  * Create resilient workflows in the existence of faults.
- * Overcome most scheduler resource time limitations.
+ * Overcome most scheduler resource time limitations, and efficiently use the lowpriority partition.
```

Comment:
(Minor/optional) Can quote lowpriority as `lowpriority` to make it appear distinctly.

Reply:
thanks added quotes